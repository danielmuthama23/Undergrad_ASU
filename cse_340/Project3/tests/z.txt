x, y, a, b;
{
    x=1;
    x=2;
    FOR ( a = 0; a < 10; a = a + 1; ) {
        SWITCH a {
            CASE 0: {
                x = x+1;  output x;
                y = y+1;
            }
            CASE 1: {
                x = x+2;  output x;
                y = y+2;
            }
            CASE 2: {
                x = x+2;  output x;
                y = y+2;
            }
            DEFAULT: {
                x = x+3;  output x;
                y = y+3;
            }
        }
          SWITCH a {
            CASE 0: {
                x = x+1;  output x;
                y = y+1;
            }
            CASE 1: {
                x = x+2;  output x;
                y = y+2;
            }
            CASE 2: {
                x = x+2;  output x;
                y = y+2;
            }
            DEFAULT: {
        FOR ( b = 0; b < 10; b = b + 1; ) {
        SWITCH b {
            CASE 0: {
                x = x+1;
                y = y+1;  output x;
            }
            CASE 1: {
                x = x+2;
                y = y+2;  output x;
            }
            CASE 2: {
                x = x+2;
                y = y+2;  output x;
            }
            DEFAULT: {
                x = x+3;
                y = y+3;  output x;
            }
        }
          SWITCH b {
            CASE 0: {
                x = x+1;
                y = y+1;  output x;
            }
            CASE 1: {
                x = x+2;
                y = y+2;  output x;
            }
            CASE 2: {
                x = x+2;
                y = y+2;  output x;
            }
            DEFAULT: {
                x = x+3;
                y = y+3;  output x;
            }
        }
        output x;
    }
            }
        }
        output x;
    }
}
2