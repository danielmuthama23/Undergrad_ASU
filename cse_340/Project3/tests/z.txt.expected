3 4 4 6 8 8 10 12 12 15 16 17 17 19 21 21 23 25 25 28 31 31 34 37 37 40 43 43 46 49 49 52 55 55 58 61 61 64 67 67 67 70 71 72 72 74 76 76 78 80 80 83 86 86 89 92 92 95 98 98 101 104 104 107 110 110 113 116 116 119 122 122 122 125 126 127 127 129 131 131 133 135 135 138 141 141 144 147 147 150 153 153 156 159 159 162 165 165 168 171 171 174 177 177 177 180 181 182 182 184 186 186 188 190 190 193 196 196 199 202 202 205 208 208 211 214 214 217 220 220 223 226 226 229 232 232 232 235 236 237 237 239 241 241 243 245 245 248 251 251 254 257 257 260 263 263 266 269 269 272 275 275 278 281 281 284 287 287 287 290 291 292 292 294 296 296 298 300 300 303 306 306 309 312 312 315 318 318 321 324 324 327 330 330 333 336 336 339 342 342 342 345 346 347 347 349 351 351 353 355 355 358 361 361 364 367 367 370 373 373 376 379 379 382 385 385 388 391 391 394 397 397 397